### PR TITLE
Xiaoxu/uirevamp

### PR DESCRIPTION
**Pull request by XiaoxuHU**

### CODE REVIEW

File: AngularApp/projects/app-service-diagnostics/src/app/home/components/search-results/search-results.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="search-result-container">
-  <div class="search-result" *ngFor="let feature of features" (click)="navigateToFeature(feature)"
```
Comment (cindylovescoding) [inline AngularApp/projects/app-service-diagnostics/src/app/home/components/search-results/search-results.component.html]: Why do we want to change this?

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html [hunk 2]
```diff
@@ -18,13 +20,16 @@ <h3 class="category-title">Availability and performance</h3>
     <span class="fa status-icon fa-info-circle" style="margin-left: 5px;"></span>
   </div>
 
-  <div class="search-box">
+  <div class="search-box" style="margin-bottom: 25px;">
```
Comment (cindylovescoding) [inline AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html]: Can you add these styles into css class?
Same as below

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html [hunk 3]
```diff
@@ -18,13 +20,16 @@ <h3 class="category-title">Availability and performance</h3>
...
     <fab-search-box placeholder="Search"></fab-search-box>
   </div>
 
-  <health-check-v3></health-check-v3>
-  <p style="font-size: 12px;">Are you looking for something else ?</p>
-  <div class="search-box">
-    <fab-search-box placeholder="Search"></fab-search-box>
+  <!-- <health-check-v3></health-check-v3> -->
```
Comment (cindylovescoding) [inline AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html]: Remove comments

File: AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        (click)="onClickCard(summaryCard)"
+        [ngClass]="{'card-container-hover':clickable[i]}"
+    >
+      <h3 class="card-title">{{summaryCard.title}}</h3>
+      <p class="card-message">
+          <span class="fa status-icon" [class.fa-exclamation-circle]="summaryCard.status === SummaryStatus.Critical"
+              [class.fa-exclamation-triangle]="summaryCard.status === SummaryStatus.Warning"
+              [class.fa-check-circle]="summaryCard.status === SummaryStatus.Success"
+              [class.fa-info-circle]="summaryCard.status === SummaryStatus.Info"
+              style="margin-right: 7px;"
```
Comment (cindylovescoding) [inline AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.html]: Same here

File: AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    });
+  }
+
+  //Todo: Add Tool type Navigation
+  onClickCard(card: SummaryCard): void {
+    switch (card.actionType) {
+      case ActionType.Detector:
+        this.navigateToDetector(card);
+        break;
+      case ActionType.None:
```
Comment (cindylovescoding) [inline AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts]: Why do we need ActionType.None?

File: AngularApp/projects/diagnostic-data/src/lib/models/detector.ts [hunk 6]
```diff
@@ -102,7 +103,8 @@ export enum RenderingType {
     ChangeAnalysisOnboarding,
     ChangesView,
     ApplicationInsightsView,
-    DependencyGraph
+    DependencyGraph,
+    SummaryCard = 22
```
Comment (cindylovescoding) [inline AngularApp/projects/diagnostic-data/src/lib/models/detector.ts]: Could you remove "= 22"? Incase other people insert some other type in between

**Comment by cindylovescoding** [comment]
Also @XiaoxuHu could you help resolve the conflicts in this PR?

File: AngularApp/projects/diagnostic-data/src/lib/models/detector.ts (hunk 6 above)
Comment (XiaoxuHU) [reply AngularApp/projects/diagnostic-data/src/lib/models/detector.ts]: No, I need to specify the SummaryCard = 22 to match the rendering type populated from our backend

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html (hunk 2 above)
Comment (XiaoxuHU) [reply AngularApp/projects/app-service-diagnostics/src/app/shared/components/category-overview/category-overview.component.html]: Sure

File: AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.html (hunk 4 above)
Comment (XiaoxuHU) [reply AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.html]: Yes

File: AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts (hunk 5 above)
Comment (XiaoxuHU) [reply AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts]: When ActionType is None, then SummaryCard will be unclickable

File: AngularApp/projects/diagnostic-data/src/lib/models/detector.ts (hunk 6 above)
Comment (cindylovescoding) [reply AngularApp/projects/diagnostic-data/src/lib/models/detector.ts]: you are hard code 22 in the backend?

File: AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts (hunk 5 above)
Comment (cindylovescoding) [reply AngularApp/projects/diagnostic-data/src/lib/components/summary-cards/summary-cards.component.ts]: I mean default is unclickable in your logic here

**Pull request merged**